ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -489,6 +528,8 @@ data MapKind = DiffMK
              | SeqDiffMK
              | TrackingMK
              | ValuesMK
+             | LMDBMK
+             | NamesMK
```

Comment:
Minor: I suggest `NameMK` instead of the plural. Thought process: it's one name per table.